app/src/renderer/components/common/PagePreferences.vue

```diff
@@ -5,16 +5,6 @@
       <tool-bar />
     </template>
     <tm-part title="Settings">
-      <tm-list-item type="field" title="Select network to connect to">
```

Comment:
Although this has to be killed here, I liked the idea of being able to switch easily between networks. When we will be on the web maybe we could recycle something from here

Reply:
I like the idea, specially if we support multiple testnets/chains